airflow/operators/trigger_dagrun.py

```diff
@@ -218,7 +221,7 @@ def execute(self, context: Context):
                 raise e
         if dag_run is None:
             raise RuntimeError("The dag_run should be set here!")
-        # Store the execution date from the dag run (either created or found above) to
+        # Store the run id from the dag run (either created or found above) to
         # be used when creating the extra link on the webserver.
```

Comment:
We should add a todo note here that execution date is stored only for backward compatibility and should be removed in Airflow 3.

Reply:
Ok.  Update pushed.